src/Entity/DrivingSchool.php

```diff
@@ -16,12 +19,37 @@ class DrivingSchool
     private ?int $id = null;
 
     #[ORM\Column(length: 255)]
+    #[Assert\NotBlank(message: 'Veuillez renseigner le nom de votre auto-école')]
+    #[Assert\Length(
+        min: 15,
+        max: 255,
+        minMessage: 'Le nom de votre auto-école doit contenir au moins {{ limit }} caractères, le votre en contient {{ value_length }}.',
+        maxMessage: 'Le nom de votre auto-école ne peut pas dépasser {{ limit }} caractères, le votre en contient {{ value_length }}.'
+    )]
     private ?string $name = null;
 
+
     #[ORM\Column(length: 255)]
+    #[Assert\NotBlank(message: 'Veuillez renseigner l\'adresse de votre auto-école')]
+    #[Assert\Length(
+        min: 10,
```

Comment:
Same as drivingSchool, maybe its too big and just need to check if null or not
Ex : Rue Pavée (9 caract)